lib/create-append-anything/append-menu/AppendContextPadProvider.js

```diff
@@ -24,7 +23,7 @@ AppendContextPadProvider.$inject = [
   'contextPad',
   'popupMenu',
   'translate',
-  'canvas'
+  'canvas',
```

Comment:
This is unnecessary

Reply:
Indeed, unnecessary things were added because of the vscode one-click fix